generic/tests/cfg/netperf.cfg

```diff
@@ -77,6 +79,7 @@
     #    log_guestinfo_script = scripts/rh_perf_log_guestinfo_script.bat
     #    log_guestinfo_exec = cmd /c
     #    log_guestinfo_path = C:\log_guestinfo.bat
+        cpu_model_flags = ",hv_time,hv_relaxed,hv_vapic,hv_spinlocks=0xfff"
```

Comment:
@qunfang Would you help confirm if ppc64le has same configuration? Thanks.

Reply:
update done, please help review. Thanks,